.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "girder",
+    "rules": {
+      "backbone/defaults-on-top": 2,
+      "backbone/event-scope": 2,
+      "backbone/no-changed-set": 2,
+      "backbone/no-collection-models": 2,
+      "backbone/no-el-assign": 2,
+      "backbone/no-native-jquery": [2, "selector"],
+      "backbone/no-silent": 2,
+      "backbone/no-view-model-attributes": 2,
+      "backbone/no-view-onoff-binding": 2,
+      "backbone/no-view-qualified-jquery": 2,
+      "backbone/render-return": 2
+    },
+    "globals": {
+        "isic": true
+    }
+}
```

Comment:
It looks like we have to specify our custom bases to apply the backbone rules to our classes:
````
    ...
    "settings": {
        "backbone": {
            "Collection": [
                "Collection"
            ],
            "Model": [
                "Model"
            ],
            "View": [
                "View"
            ]
        }
    }
````